ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml

```diff
@@ -44,6 +44,9 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+test_binary_build_commands: mkdir -p /usr/local/bin && curl -Lfso /usr/local/bin/jq
+  https://github.com/stedolan/jq/releases/download/jq-1.7.1/jq-linux64 && chmod +x
+  /usr/local/bin/jq
```

Comment:
This doesn't seem right split over multiple lines? Shouldn't this be a single line?

Reply:
Also, the indentation seems wrong.

Reply:
> This doesn't seem right split over multiple lines? Shouldn't this be a single line?
the command gets autoformatted across multiple lines after `make ci-operator-config`, I've split it out to separate lines.
> Also, the indentation seems wrong.
I think that was also the `make ci-operator-config` output, the  `pj rehearse jobs` run without issues with the indentation.

Reply:
@tmshort @ankitathomas Have we resolved this or it is still outstanding?

Reply:
it's resolved. I'll mark it so there's no confusion.